compiler/types/Type.hs

```diff
@@ -2273,7 +2318,7 @@ nonDetCmpTypeX env orig_t1 orig_t2 =
       | Just (s1, t1) <- repSplitAppTy_maybe ty1
       = go env s1 s2 `thenCmpTy` go env t1 t2
     go env (FunTy w1 s1 t1) (FunTy w2 s2 t2)
-      = liftOrdering (compare w1 w2) `thenCmpTy`
+      = --liftOrdering (compare w1 w2) `thenCmpTy` TODO: MattP need to add this back in
```

Comment:
Complete your comment explaining why it is commented out and what it does, maybe?